java/ql/src/semmle/code/java/frameworks/ratpack/RatpackHttp.qll

```diff
@@ -0,0 +1,59 @@
+/**
+ * Provides classes and predicates related to `ratpack.http.*`.
+ */
+
+import java
+
+/**
+ * The interface `ratpack.http.Request`.
+ * https://ratpack.io/manual/current/api/ratpack/http/Request.html
+ */
+library class RatpackRequest extends RefType {
```

Comment:
If I remember correctly `library` is neither needed nor used anymore.

Reply:
Yeah, it's obsolete -- use `private` if applicable